include/mgp.hpp

```diff
@@ -570,7 +580,6 @@ class List {
   /// @brief returns the string representation
   std::string ToString() const;
 
- private:
```

Comment:
Not the right solution. I have to figure out how to get `mgp_list` from `mgp::List`.